actions/setup_e2e_tests.sh

```diff
@@ -8,20 +8,20 @@ BRANCH=`echo ${VERSION} | cut -d "." -f1-2`
 DEBTEST=`lsb_release -a 2> /dev/null | grep Distributor | awk '{print $3}'`
 RHTEST=`cat /etc/redhat-release 2> /dev/null | sed -e "s~\(.*\)release.*~\1~g"`
 
+# Decrease interval for MongoDB TTL expire thread. By default it runs every 60 seconds which
+# means we would need to wait at least 60 seconds in our key expire end to end tests.
+# By decreasing it, we can speed up those tests
+# TODO: Use db.adminCommand, but for that we need to fix admin user permissions in bootstrap script
+echo -e "\nsetParameter:\n  ttlMonitorSleepSecs: 1" >> /etc/mongod.conf
```

Comment:
NOTE: Ideally we could use ``mongo --eval`` and ``db.adminCommand`` here, but it turns out the admin user we create during installer script run doesn't actually give root permissions to admin user so it doesn't have access to run this command...

Reply:
🤦‍♂